openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardNew/CommentCard.component.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { Tooltip, Typography } from 'antd';
+import classNames from 'classnames';
+import { compare } from 'fast-json-patch';
+import React, { useCallback, useMemo, useState } from 'react';
+import { Thread } from '../../../generated/entity/feed/thread';
+import {
+  formatDateTime,
+  getRelativeTime,
+} from '../../../utils/date-time/DateTimeUtils';
+import {
+  getFrontEndFormat,
+  MarkdownToHTMLConverter,
+} from '../../../utils/FeedUtils';
+import ProfilePicture from '../../common/ProfilePicture/ProfilePicture';
+import FeedCardFooterNew from '../ActivityFeedCardV2/FeedCardFooter/FeedCardFooterNew';
+import ActivityFeedEditor from '../ActivityFeedEditor/ActivityFeedEditorNew';
+import { useActivityFeedProvider } from '../ActivityFeedProvider/ActivityFeedProvider';
+import ActivityFeedActions from '../Shared/ActivityFeedActions';
+
+const { Text } = Typography;
+
+interface CommentCardInterface {
+  feed: Thread;
+  post: any;
+  isLastReply: boolean;
+}
+
+const CommentCard = ({ feed, post, isLastReply }: CommentCardInterface) => {
+  const { updateFeed } = useActivityFeedProvider();
+  const [isHovered, setIsHovered] = useState(false);
+  const [isEditPost, setIsEditPost] = useState<boolean>(false);
+  const [postMessage, setPostMessage] = useState<string>('');
+  const seperator = '.';
+
+  const onEditPost = () => {
+    setIsEditPost(!isEditPost);
+  };
+
+  const onUpdate = (message: string) => {
+    const updatedPost = { ...feed, message };
+    const patch = compare(feed, updatedPost);
+    updateFeed(feed.id, post.id, false, patch);
```

Comment:
Don't we need to await here?
Make sure to await on all the API calls

Reply:
updateFeed is not returning promise. Even in our legacy code for ActivityFeedCardv2 we have not used await while calling updateFeed